Rubberduck.Core/AutoComplete/AutoCompleteBlockBase.cs

```diff
@@ -30,44 +30,51 @@ protected AutoCompleteBlockBase(IConfigProvider<IndenterSettings> indenterSettin
 
         public override bool Execute(AutoCompleteEventArgs e)
         {
-            if (SkipPreCompilerDirective && e.OldCode.Trim().StartsWith("#"))
+            if (e.Keys != System.Windows.Forms.Keys.Enter /*&& e.Keys != System.Windows.Forms.Keys.Tab*/)
```

Comment:
This looks unfinished.

Reply:
Yeah I'm not sure I want to implement TAB completion in this PR. I'll remove it.